iXBRLViewerPlugin/viewer/src/js/ixbrlviewer.js

```diff
@@ -19,9 +19,9 @@ import { Viewer } from "./viewer.js";
 import { Inspector } from "./inspector.js";
 
 export function iXBRLViewer() {
-    this._plugins = [];
-    this.inspector = new Inspector(this);
-    this.viewer = null;
+	this._plugins = [];
+	this.inspector = new Inspector(this);
+	this.viewer = null;
```

Comment:
The rest of the repo uses spaces, so this change to tabs seems wrong.

Reply:
Fixed in a0ef97d